addons/web/static/src/views/relational_model.js

```diff
@@ -745,7 +745,10 @@ export class Record extends DataPoint {
             this.activeFields[fieldName].domain,
         ];
 
-        const evalContext = this.evalContext;
+        const evalContext = Object.assign(
```

Comment:
I don't think you wanna overwrite this.context
I guess ```
Object.assign({}, this.context, this.evalContext);
``` is more correct

Reply:
Done :+1:
I'll leave it to you to add a good test covering the problem ? :pray: